include/float.h

```diff
@@ -0,0 +1,123 @@
+/* <float.h> for the Aro C compiler */
+
+#pragma once
+
+#undef FLT_RADIX
+#define FLT_RADIX __FLT_RADIX__
+
+#undef FLT_MANT_DIG
+#define FLT_MANT_DIG __FLT_MANT_DIG__
+
+#undef DBL_MANT_DIG
+#define DBL_MANT_DIG __DBL_MANT_DIG__
+
+#undef LDBL_MANT_DIG
+#define LDBL_MANT_DIG __LDBL_MANT_DIG__
+
+#if __STDC_VERSION__ >= 199901L
+#undef DECIMAL_DIG
+#define DECIMAL_DIG __DECIMAL_DIG__
+#endif  // __STDC_VERSION__ >= 199901L
```

Comment:
The preprocessor should just ignore the extra tokens but it'd still be better to use C style `/* */` comments in these headers.